server/lib/SMS-sender.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const { SNSClient, PublishCommand } = require('@aws-sdk/client-sns');
+const config = require('./config');
+
+/**
+ * This is configured SNSClient that is used with AWS-SNS API.
+ * Contains an error if something went wrong during initialization.
+ * This constant is automatically initialized.
+ * @type {SNSClient}
+ */
+const client = function() {
+    try {
+        process.env.AWS_ACCESS_KEY_ID = config.SMS.keyID;
+        process.env.AWS_SECRET_ACCESS_KEY = config.SMS.secretKey;
+        return new SNSClient({region: config.SMS.region});
+    }
+    catch (error) {
+        return error;
```

Comment:
This is intended. Silent error is good when a user does not want to use this feature. It returns status information. If a calling function cares about the result, it can read it and then rethrow or log or do something appropriate.

Reply:
Then do it as opt-in feature. In config disable it by default for example and if disabled let it fail silently. But this way you will always just get TypeError on send, as client will be string or error object without `send`. This will in no way indicate what the problem is.